backend/controllers/searchExternalUsers.test.js

```diff
@@ -72,69 +89,83 @@ describe('search factory', () => {
   })
 
   it('should call search user render', async () => {
-    const req = { params: {}, flash: jest.fn() }
-    getAssignableGroupsApi.mockResolvedValue([{ text: 'name', value: 'code' }])
-    getSearchableRolesApi.mockResolvedValue([{ roleName: 'name', roleCode: 'code' }])
-
+    const req = {
+      ...standardReq,
+    }
+    mockGroupsAndRolesCalls()
+    searchApi.mockResolvedValue(noUsers)
     const render = jest.fn()
-    await search.index(req, { render })
+    await search(req, { render })
+
     expect(render).toBeCalledWith('searchExternalUsers.njk', {
       searchTitle: 'Search for an external user',
       searchUrl: '/search-external-users',
-      groupDropdownValues: [{ text: 'name', value: 'code' }],
-      roleDropdownValues: [{ text: 'name', value: 'code' }],
+      maintainUrl: '/manage-external-users',
+      pagination: undefined,
+      results: [],
+      groupDropdownValues: [{ text: 'groupName', value: 'group_code' }],
+      roleDropdownValues: [{ text: 'roleName', value: 'role_code' }],
+      currentFilter: {
+        groupCode: undefined,
+        roleCode: undefined,
+        status: 'ALL',
+        user: undefined,
+      },
+      downloadUrl: undefined,
+      errors: undefined,
     })
   })
 
   describe('results', () => {
     it('should call external search results render', async () => {
       const req = {
+        ...standardReq,
         query: { user: 'joe' },
         flash: jest.fn(),
         get: jest.fn().mockReturnValue('localhost'),
         protocol: 'http',
-        originalUrl: '/results',
```

Comment:
no longer have a separate url as all search filtering and results are on the same page